tools/ExamplesGenerator.ps1

```diff
@@ -0,0 +1,268 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+Param(
+    $ModulesToGenerate = @(),
+    [string] $ModuleMappingConfigPath = (Join-Path $PSScriptRoot "..\config\ModulesMapping.jsonc")
+)
+function Get-GraphMapping {
```

Comment:
You can safely delete this function since `$GraphMapping` in `Start-Generator` does what this function does.